App/frontend/src/components/styling/responsive/_registerpage.scss

```diff
@@ -0,0 +1,122 @@
+.registerPage{
+
+    .page-content{
+    @include media('>phone', '>=520px'){
+        display:flex;
+        flex-direction: row;
+        height:100vh;
+        
+        
+    }
+    }
+
+    .sign-in-part{
+        @include media('>phone', '>=520px'){
```

Comment:
And i also have another suggestion, which is that you can clean up this scss code a little by deleting most of the @include media() lines.

Reply:
for example u can clean up some of the code by doing something like this:

.registerPage{
    .page-content{
        @include media('>phone', '>=520px'){
            display:flex;
            flex-direction: row;
            height:100vh;
            
            .sign-in-part{
                width:40%;
                height:100vh;

                h1 {
                    margin-top:2.1em;
                }
            }
            
            .register-part{
                width:60%;
                height:100vh;
        
                h1{
                    margin-top: 6.8em;
                }
            }
        }
    }
}

same with the other unnecessary lines

Reply:
@theotrg07 that would be all from me for this time, in any case great job, ur improving a lot

Reply:
But dont we need to do something with bigger screen sizes when the screen seems kinda empty and the input fields are a little bit too big?

Reply:
oh yeah ur right, but we will leave it in there for now, since we do not have anything in mind for it, we'll just skip it

Reply:
> And i also have another suggestion, which is that you can clean up this scss code a little by deleting most of the @include media() lines.

Ok i did it

Reply:
tell me if there's something else, if not, i will close this conversation and pull request